cmds/core/tty/tty.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"syscall"
+)
+
+func run(stdin io.Writer) error {
+	fi, err := os.Stdin.Stat()
```

Comment:
did you want to use stdin here, or os.Stdin?

Reply:
I want os.Stdin, stdin renamed to stdout.